src/main/kotlin/uk/gov/dluhc/printapi/database/repository/AnonymousElectorDocumentSummaryRepository.kt

```diff
@@ -1,29 +1,12 @@
 package uk.gov.dluhc.printapi.database.repository
 
-import org.springframework.data.domain.Page
-import org.springframework.data.domain.Pageable
 import org.springframework.data.jpa.repository.JpaSpecificationExecutor
 import org.springframework.data.repository.PagingAndSortingRepository
 import org.springframework.stereotype.Repository
 import uk.gov.dluhc.printapi.database.entity.AnonymousElectorDocumentSummary
-import uk.gov.dluhc.printapi.database.entity.SourceType
 import java.util.UUID
 
 @Repository
 interface AnonymousElectorDocumentSummaryRepository :
     PagingAndSortingRepository<AnonymousElectorDocumentSummary, UUID>,
-    JpaSpecificationExecutor<AnonymousElectorDocumentSummary> {
-
-    fun findAllByGssCodeInAndSourceType(
```

Comment:
These methods are no longer necessary or used, now that the service class uses  the repository `findAll` method with the build specification and pageable 👍